### DISCUSSION TITLE
[入驻]大家好我是丛林魔君亚力姆，有什么灾厄相关问题可以向我咨询

### DISCUSSION THREAD

**Discussion by Griffin233** [share121/inter-knot#884] [其他]
如有发现奴役蜜蜂者速速向我报告，奴役蜜蜂属实可耻
![Uploading 4929b630d6c64c97a661842e8b380f6.jpg…]()

**Comment by sky27950** [comment]
魔君，嘉登背着你把蜂王改造成瘟疫使者了
